starlite/connection/base.py

```diff
@@ -260,7 +261,7 @@ def cache(self) -> "Cache":
         """
         return self.app.cache
 
-    def set_session(self, value: Union[Dict[str, Any], "BaseModel"]) -> None:
+    def set_session(self, value: Union[Dict[str, Any], "BaseModel", Type["Empty"]]) -> None:
```

Comment:
Why `Type["Empty"]`? Shouldn't be `EmptyType` instead?

Reply:
It should be, good catch!